src/index.html

```diff
@@ -7,6 +7,333 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a
+          href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__link"
+              href="#apple"
+            >
```

Comment:
you should move attributes in the new line in 2 cases:
1. your line is more than max allowable line
2. you have 3 or more attrs